File: src/main/java/com/AlphaStore/Utils/UtilsApplication.java
package com.AlphaStore.Utils;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@SpringBootApplication
@EnableDiscoveryClient
public class UtilsApplication {

	public static void main(String[] args) {
		SpringApplication.run(UtilsApplication.class, args);
	}

}

File: src/main/kotlin/com/AlphaStore/Utils/UtilsApplication.kt
package com.alphaStore.Utils

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication(
	scanBasePackages = [
		"com.alphaStore.utils",
		"com.alphaStore.core",
	]
)
@EnableDiscoveryClient
class UtilsApplication

fun main(args: Array<String>) {
	runApplication<UtilsApplication>(*args)
}
